core/test/input/optimizers/test.cfg

```diff
@@ -0,0 +1,114 @@
+############ Spirit Configuration ###############
+
+################## General ######################
+output_tag_time   1
+log_to_console    0
+log_to_file       1
+log_file_level    6
+################## End General ##################
+
+################## Geometry #####################
+### Read basis from file
+basis
+1.0 0.0 0.0
+0.0 1.0 0.0
+0.0 0.0 1.0
+1
+0 0 0
+
+### Keyword translation_vectors ###
+###   t1.x t1.y t1.z nCells(t1)
+###   t2.x t2.y t2.z nCells(t2)
+###   t3.x t3.y t3.z nCells(t3)
+translation_vectors
+1 0 0 15
+0 1 0 15
+0 0 1 1
+################# End Geometry ##################
+
+################## Hamiltonian ##################
+
+### Hamiltonian Type (heisenberg_neighbours, heisnberg_pairs, gaussian )
+hamiltonian   heisenberg_neighbours
+
+### boundary_conditions (in a b c) = 0(open), 1(periodical)
+boundary_conditions 1 1 0
+
+### external magnetic field vector[T]
+external_field_magnitude  1.0
+external_field_normal     0.0 0.0 1.0
+
+### µSpin
+mu_s    1.0
+
+### Uniaxial anisotropy constant [meV]
+anisotropy_magnitude    0.0
+anisotropy_normal       0.0 0.0 1.0
+
+### Exchange constants [meV] for the respective shells
+### Jij should appear after the >Number_of_neighbour_shells<
+n_neigh_shells_exchange   2
+jij                       00.0
+
+### Chirality of DM vectors (+/-1=bloch, +/-2=neel)
+dm_chirality    2
+
+### DM constant [meV]
+n_neigh_shells_dmi  1
+dij                 6.0
+
+### Dipole-Dipole radius
+dd_radius   0.0
+
+################ End Hamiltonian ################
+
+############ Method Output ######################
+
+llg_output_any     1    # Write any output at all
```

Comment:
set this to 0 (we don't need output here) and leave the other _output_ parameters unspecified.